docs/specification.md

```diff
@@ -292,9 +292,9 @@ The "Merge Contributors Modal" includes:
 
 The "Profile" page includes:
 
-- "Info" block: Includes prefilled “Name” and “Email” Fields and “Update Profile” button. The “Email” field is disabled. The “Update Profile” button becomes active after the user changes fields. When clicked, profile data should be updated.
-- “Password” block: Includes “Password” and “Confirm password” fields and “Update Password” button. The “Update Password” button becomes active after the user inputs something into the “Password” field. After submission user password should be updated.
-- “Delete account” block: Includes “Delete your account” button. When the “Delete your account” button is clicked, a confirmation modal is opened. After confirming the action, the account should be deleted as well as data related to it.
+- "Info" block: Includes prefilled “Name” and “Email” Fields and “Update Profile” button. The “Email” field is disabled. The “Update Profile” button becomes active after the user changes fields. When clicked, profile data should be updated. After updating the profile data, the message "Your profile has been successfully updated" appears.
```

Comment:
The messages should end with a period.

Reply:
I fixed in [commit](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/98/commits/7e92d13fd4dedc978d68f7bc2a116f8e5f0fa71c)